Quantum-Control-Applications-QuAM/Superconducting/quam_builder/builder/qop_connectivity/build_quam_wiring.py

```diff
@@ -1,26 +1,26 @@
-import os
-from pathlib import Path
 from typing import Union
 from qualang_tools.wirer import Connectivity
-from quam_libs.components.superconducting.qpu import BaseQuAM, FixedFrequencyQuAM, FluxTunableQuAM
-from quam_libs.quam_builder.qop_connectivity.create_wiring import create_wiring
+from quam_builder.architecture.superconducting.qpu import AnyQuAM
+from quam_builder.builder.qop_connectivity.create_wiring import create_wiring
 from quam.components.ports import (
     FEMPortsContainer,
     OPXPlusPortsContainer,
 )
 
+
 def build_quam_wiring(
-    connectivity: Connectivity, host_ip: str, cluster_name: str, quam_state_path: Union[Path, str], quam_instance: Union[BaseQuAM, FixedFrequencyQuAM, FluxTunableQuAM], port: int = None
+    connectivity: Connectivity, host_ip: str, cluster_name: str,
+    quam_instance: AnyQuAM, port: int = None
 ):
 
     machine = quam_instance
     add_ports_container(connectivity, machine)
     add_name_and_ip(machine, host_ip, cluster_name, port)
     machine.wiring = create_wiring(connectivity)
-    save_machine(machine, quam_state_path)
+    save_machine(machine)
```

Comment:
How about we remove `save_machine`? If we want special save behaviour, how about we override `QUAM.save()`?

Reply:
yes let's do this in another PR